### PR TITLE
Adding restart network virtual appliance powershell cmdlets

### PR DESCRIPTION
**Pull request by arsheen**
## Description

This cmdlet will be used by customers to restart the network virtual appliance vm instances. It is similar to the restart vmss powershell command let.
Approved design review: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1433

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Network/Network/Models/PSNetworkVirtualApplianceRestartOperationStatusResponse.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿using System;
```
Comment (NoriZC) [inline src/Network/Network/Models/PSNetworkVirtualApplianceRestartOperationStatusResponse.cs:1]: Please follow this [example](https://github.com/Azure/azure-powershell/blob/8dddb77e2773b9a6778f8e9c91c6a9876d40ae1f/src/Network/Network/Models/PSApplicationGatewayFirewallCustomRuleGroupByUserSession.cs#L2)  to add copyright for newly created files.

Comment (NoriZC) [inline src/Network/Network/Models/PSNetworkVirtualApplianceRestartOperationStatusResponse.cs:1]: Please follow this [example](https://github.com/Azure/azure-powershell/blob/8dddb77e2773b9a6778f8e9c91c6a9876d40ae1f/src/Network/Network/Models/PSApplicationGatewayFirewallCustomRuleGroupByUserSession.cs#L2)  to add copyright for newly created files.

Comment (arsheen) [reply src/Network/Network/Models/PSNetworkVirtualApplianceRestartOperationStatusResponse.cs]: Done

**Comment by arsheen** [comment]
@NoriZC : PR updated with suggested changes. All checks are now green.

**Comment by arsheen** [comment]
Hi @NoriZC , could you please merge this PR for me? I do not have the authorization to merge this PR into the branch.

**Pull request merged**